specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2024-04-02-preview/managedClusters.json

```diff
@@ -5117,6 +5139,10 @@
           "title": "The IP families used to specify IP versions available to the cluster.",
           "description": "IP families are used to determine single-stack or dual-stack clusters. For single-stack, the expected value is IPv4. For dual-stack, the expected values are IPv4 and IPv6."
         },
+        "podLinkLocalAccess": {
+          "$ref": "#/definitions/PodLinkLocalAccess",
+          "description": "Defines access to special link local addresses (IMDS and wireserver) for pods with hostNetwork=false"
```

Comment:
Minor. Should probably include here, "if not specified, the default is "IMDS"?

Reply:
Thanks for reviewing! Updated.